webclient/src/types/server.tsx

```diff
@@ -54,6 +54,12 @@ export class Host {
 }
 
 export const DefaultHosts: Host[] = [
+  {
+    name: 'Chickatrice',
+    host: 'mtg.chickatrice.net/servatrice',
+    port: '443',
```

Comment:
This seemed to work for me, testing on beta (wss) and a locally running instance (ws).

```suggestion
    host: 'mtg.chickatrice.net',
    port: '443',
    localPort: '4748
```